schemas/com.snowplowanalytics.snowplow/ua_parser/ua_parser_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
^^ This path is wrong. Can we `git mv` to:

    schemas/com.snowplowanalytics.snowplow/ua_parser_config/jsonschema/1-0-0